packages/web/src/components/audio-transactions-table/AudioTransactionsTable.tsx

```diff
@@ -103,18 +107,21 @@ const renderDateCell = (cellInfo: TransactionCell) => {
 
 const renderChangeCell = (cellInfo: TransactionCell) => {
   const tx = cellInfo.row.original
-  const isPositive = isChangePositive(tx)
   const { change } = tx
   return (
-    <Tooltip text={`${formatAudio(tx.change, 2)} $AUDIO`} mount={'body'}>
+    <Tooltip
+      text={`${wAUDIO(BigInt(tx.change)).toFixed(2)} $AUDIO`}
```

Comment:
nit: probably better to do `toLocaleString(.., { minimumFractionDigits: 2, maximumFractionDigits: 2 })` to get the grouping etc unless we don't want that?